warehouse/templates/packaging/detail.html

```diff
@@ -90,6 +90,7 @@ <h1 class="package-header__name">{{ release.project.name }} {{ release.version }
   </div>
 </section>
 
+{% if release.urls.values()|filter_valid_uris|length > 0 %}
```

Comment:
Ok, I guess. I did it this way for robustness, but your way is probably better for the short term.